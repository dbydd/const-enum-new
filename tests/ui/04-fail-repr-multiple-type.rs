#![feature(const_trait_impl)]

use const_enum::ConstEnum;

#[derive(ConstEnum)]
#[repr(u8, u8)]
enum Test {
    A = 1,
    B = 2,
    C = 3,
}

pub fn main() {}
