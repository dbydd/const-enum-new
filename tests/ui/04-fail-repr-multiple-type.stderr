error: proc-macro derive panicked
 --> tests/ui/04-fail-repr-multiple-type.rs:5:10
  |
5 | #[derive(ConstEnum)]
  |          ^^^^^^^^^
  |
  = help: message: malformed repr attribute, expected single type

error[E0566]: conflicting representation hints
 --> tests/ui/04-fail-repr-multiple-type.rs:6:8
  |
6 | #[repr(u8, u8)]
  |        ^^  ^^
  |
  = note: `#[deny(conflicting_repr_hints)]` on by default
  = warning: this was previously accepted by the compiler but is being phased out; it will become a hard error in a future release!
  = note: for more information, see issue #68585 <https://github.com/rust-lang/rust/issues/68585>
